doc/cliques.xml

```diff
@@ -203,7 +219,7 @@ gap> DigraphClique(gr, [], [], 2);
     exclude[, limit[, size]]]]"/>
   <Attr Name="DigraphMaximalCliquesAttr" Arg="digraph"/>
   <Attr Name="DigraphMaximalCliquesRepsAttr" Arg="digraph"/>
-  <Returns>A list of lists of positive integers.</Returns>
+  <Returns>An immutable list of immutable lists of positive integers.</Returns>
```

Comment:
Technically the second immutable here is redundant, since an immutable object can't contain mutable ones (as far as I'm aware).  But never mind.